src/Docker.DotNet/Endpoints/StreamUtil.cs

```diff
@@ -10,37 +10,44 @@ namespace Docker.DotNet.Models
 {
     internal static class StreamUtil
     {
-        private static Newtonsoft.Json.JsonSerializer _serializer = new Newtonsoft.Json.JsonSerializer();
+        private static readonly Newtonsoft.Json.JsonSerializer _serializer = new Newtonsoft.Json.JsonSerializer();
 
         internal static async Task MonitorStreamAsync(Task<Stream> streamTask, DockerClient client, CancellationToken cancel, IProgress<string> progress)
         {
+            var tcs = new TaskCompletionSource<string>();
+            cancel.Register(() => tcs.TrySetCanceled(default));

```

Comment:
Why not in a using like the others?

Reply:
Should be like others by next commit. I'm running tests now before pushing again